ehr/src/org/labkey/ehr/buttons/CageBulkEditButton.java

```diff
@@ -32,9 +34,9 @@ public CageBulkEditButton(Module owner)
     {
         super(owner, "Bulk Edit Cage/Divider Types", "EHR.window.BulkEditCageTypeWindow.buttonHandler(dataRegionName);");
 
-        LinkedHashSet cds = new LinkedHashSet<ClientDependency>();
-        cds.add(ClientDependency.fromModuleName("ehr"));
-        cds.add(ClientDependency.fromPath("ehr/window/BulkEditCageTypeWindow.js"));
+        List<Supplier<ClientDependency>> cds = new ArrayList<>();
+        cds.add(ClientDependency.supplierFromModuleName("ehr"));
+        cds.add(ClientDependency.supplierFromPath("ehr/window/BulkEditCageTypeWindow.js"));
         setClientDependencies(cds);
```

Comment:
`setClientDependencies(List.of(...))` would be more succinct. Just a suggestion...